pesacheck_meedan_bridge/.env.example

```
@@ -0,0 +1,6 @@
+PESACHECK_URL=https://pesacheck.org/feed
+PESACHECK_RSS_2_JSON_API_KEY=
```

```suggestion
PESACHECK_RSS2JSON_API_KEY=
```
I'm assuming `RSS2JSON` is a single name.